src/main/java/com/hubspot/jinjava/el/ExpressionResolver.java

```diff
@@ -72,7 +69,9 @@ public Object resolveExpression(String expression) {
       String elExpression = EXPRESSION_START_TOKEN + expression.trim() + EXPRESSION_END_TOKEN;
       ValueExpression valueExp = expressionFactory.createValueExpression(elContext, elExpression, Object.class);
       Object result = valueExp.getValue(elContext);
-
+      if(result == null && interpreter.getConfig().isFailOnUnknownTokens()) {
```

Comment:
space after `if` please

Reply:
the problem I'm facing is that when here I do not get the lineNumber and startPosition when I'm about to throw an error here. I only have the expression string with me.

Reply:
```java
String elExpression = EXPRESSION_START_TOKEN + expression.trim() + EXPRESSION_END_TOKEN;
      ValueExpression valueExp = expressionFactory.createValueExpression(elContext, elExpression, Object.class);
      Object result = valueExp.getValue(elContext);
      if (result == null && interpreter.getConfig().isFailOnUnknownTokens()) {
        throw new UnknownTokenException(expression);
      }
```

Reply:
should I, in this case, pass -1 and -1 for the same?

Reply:
you have access to the interpreter, so you can use `interpreter.getLineNumber(), interpreter.getPosition()`

Reply:
done, thanks for the help 😄